declcache.go

```diff
@@ -224,9 +225,23 @@ func build_package(p *build.Package) error {
 		log.Printf("package object: %s", p.PkgObj)
 		log.Printf("package source dir: %s", p.Dir)
 		log.Printf("package source files: %v", p.GoFiles)
+		log.Printf("GOPATH: %v", g_daemon.context.GOPATH)
+		log.Printf("GOROOT: %v", g_daemon.context.GOROOT)
 	}
+	env := os.Environ()
+	for i, v := range env {
+		if strings.HasPrefix(v, "GOPATH=") {
+			env[i] = "GOPATH=" + g_daemon.context.GOPATH
+		} else if strings.HasPrefix(v, "GOROOT=") {
```

Comment:
That's one of my problems. I blindly accept patches for features I don't use. I don't use "autobuild", I think it's a broken feature from the beginning, so I don't really care what people do with it.